docs/tutorials/mnist.ipynb

```diff
@@ -873,32 +872,33 @@
     },
     {
       "cell_type": "code",
+      "execution_count": 27,
       "metadata": {
+        "colab": {},
         "colab_type": "code",
-        "id": "n8vuQpSLlBV2",
-        "colab": {}
+        "id": "n8vuQpSLlBV2"
       },
+      "outputs": [],
       "source": [
         "EPOCHS = 3\n",
         "BATCH_SIZE = 32\n",
+        "\n",
         "NUM_EXAMPLES = len(x_train_tfcirc)"
-      ],
-      "execution_count": 0,
-      "outputs": []
+      ]
     },
     {
       "cell_type": "code",
+      "execution_count": 28,
```

Comment:
Did you restart and clear outputs and this still hung around ?

Reply:
Fixed.